apps/omg_child_chain/lib/child_chain.ex

```diff
@@ -66,6 +67,11 @@ defmodule OMG.ChildChain do
     |> result_with_logging()
   end
 
+  defp is_supported(%Transaction.Recovered{signed_tx: %Transaction.Signed{raw_tx: raw_tx}})
+       when is_map(raw_tx) do
+    Map.get(raw_tx, :__struct__) in [Transaction.Payment]
```

Comment:
is there a better approach to accomplish the same result?

Reply:
Yeah, I thing this may be part of the `Transaction.Protocol` so we don't miss it implementing new tx types.